active_learning/liver_tumor_al/active_learning.py

```diff
@@ -0,0 +1,607 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import copy
+import logging
+import os
+import json
+import random
+import numpy as np
+import torch
+import pickle
+import matplotlib.pyplot as plt
+
+from monai.networks.nets import UNet
+from monai.networks.layers import Norm
+from monai.losses.dice import DiceCELoss
+from monai.metrics import DiceMetric
+from monai.inferers import sliding_window_inference
+from monai.utils import set_determinism
+from monai.data import DataLoader, CacheDataset, Dataset, decollate_batch
+from monai.transforms import (
+    AsDiscrete,
+    CropForegroundd,
+    Compose,
+    EnsureType,
+    EnsureTyped,
+    EnsureChannelFirstd,
+    LoadImaged,
+    Orientationd,
+    RandCropByPosNegLabeld,
+    RandShiftIntensityd,
+    RandAdjustContrastd,
+    RandScaleIntensityd,
+    RandGaussianNoised,
+    RandFlipd,
+    ScaleIntensityRanged,
+    Spacingd,
+    RandRotate90d,
+    SpatialPadd,
+)
+
+from torch.utils.tensorboard import SummaryWriter
+
+logger = logging.getLogger(__name__)
+parser = argparse.ArgumentParser(description="Active Learning Setting")
+
+# Directory & Json & Seed
+parser.add_argument("--base_dir", default="/home/vishwesh/experiments/al_sanity_test_apr27_2023", type=str)
```

Comment:
Hi @finalelement ,
     It would be better to not use private path in the tutorial. Maybe replace the `/home/vishwesh` with `/workspace/tutorials` or something without a user name.

Thanks,
Bin

Reply:
I fixed it in [f1a0e32](https://github.com/Project-MONAI/tutorials/pull/1365/commits/f1a0e32d2d203412b79efe7793967236f36b4f90)